.travis.yml

```diff
@@ -44,11 +44,6 @@ matrix:
           env:
             - SCAPY_COVERAGE=yes
 
-        - os: osx
```

Comment:
This non-root test is not useful as the default native osx support works perfectly fine for non-root usages.